CHANGES.rst

```diff
@@ -1519,7 +1519,7 @@ v21.1.0
 * #572: In build_ext, now always import ``_CONFIG_VARS``
   from ``distutils`` rather than from ``sysconfig``
   to allow ``distutils.sysconfig.customize_compiler``
-  configure the OS X compiler for ``-dynamiclib``.
```

Comment:
Does it make sense to change the entries in the changelog? After all, at the time, it was called OS X.